war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2515,6 +2515,9 @@ function buildFormTree(form) {
                 jsonElement = e;
                 continue;
             }
+            if(e.hasClassName('json-ignore')){
```

Comment:
Is there really no other element to support content that should be ignores in JSON?

Reply:
I just changed that after trying the write a first comment. This code only remove the JSON part but actually the token value is submitted. So I changed the input(type=hidden) to a simple span with style and remove this part.